nix/migration-tests.nix

```diff
@@ -20,7 +20,7 @@
 # individually. For example, run step1 to set up a state directory for
 # the old version, then try:
 #   stateDir=./debug /nix/store/...-launch-migration-test-v2019-12-16.sh step1
-#   stack exec migration-test -- step2 launch --state-dir=./debug ...
+#   exec migration-test -- step2 launch --state-dir=./debug ...
```

Comment:
Perhaps:
```suggestion
#   cabal run migration-test -- step2 launch --state-dir=./debug ...
```

Reply:
… except that we do not have a `migration-test` component in any of our .cabal files. 😅 It appears that these tests for database migration have never been ported to the Shelley era and are completely broken.

Fixing the database migration tests is out of scope for this pull request, so I would prefer to write something blatantly wrong rather than try to pretend that the command `cabal run migration-test` may have worked at some point.